src/interface/builddir.cpp

```diff
@@ -454,13 +454,40 @@ static int li_bd_installfile(lua_State *L)
 	return 0;
 }
 
+static int li_bd_ls(lua_State *L)
+{
+	if(lua_gettop(L) != 2) {
+		throw CustomException("ls() requires exactly 1 argument");
+	}
+	if(!lua_istable(L, 1)) {
+		throw CustomException("ls() must be called using : not .");
+	}
+	if(lua_isstring(L, 2) == 0) {
+		throw CustomException("ls() expects a string as the only argument");
+	}
+
+	std::string path = li_get_package()->relative_fetch_path(lua_tostring(L, 2));
```

Comment:
I think this will need some extra code to handle dealing with searching all the overlays as well

Reply:
added helper in the Package class